ci/gpu/build.sh

```diff
@@ -39,6 +39,21 @@ $CC --version
 $CXX --version
 conda list
 
+logger "Check GPU running the tests..."
+GPU="$(nvidia-smi | awk '{print $4}' | sed '8!d')"
+echo "Running tests on $GPU"
+
+if [[ $GPU == *"P100"* ]]; then
+  logger "Building for Pascal..."
+  GPU_ARCH=60
+elif [[ $GPU == *"V100"* ]]; then
+  logger "Building for Volta..."
+  GPU_ARCH=70
+elif [[ $GPU == *"T4"* ]]; then
+  logger "Building for Turing..."
+  GPU_ARCH=75
+fi 
```

Comment:
What happens if none of these match?

Reply:
AFAIK only v100, p100 and t4 are being used, right? So if new GPUs are added, this script will need to be updated. That said will add a condition to build all if none of those gpus are detected it'll build for all archs

Reply:
With my local gpuCI script on its way the types of GPUs that the script will utilize will open up to anything that RAPIDS supports. So the default option is a requirement for this use case. :)

Reply:
@dantegd Definitely add a default case which builds for all.